nexus_demos/config/zenoh/nexus_network_config.yaml

```diff
@@ -18,15 +18,12 @@ system_orchestrators:
       # TODO(luca) check if we need estop
       subscribers: ["/estop", "/.*/workcell_state"]
       service_servers: ["/list_workcells", "/register_workcell", "/register_transporter"]
-      # TODO(luca) check if transporter needs available endpoint
       service_clients: ["/.*/queue_task", "/.*/remove_pending_task", "/.*/pause", "/.*/signal", "/.*/get_state", "/.*/change_state", "/.*/is_task_doable"]
-      action_servers: []
-      # TODO(luca) check if we really need transporter client
-      action_clients: ["/.*/request", "/.*/transport"]
+      action_servers: ["/bid_transporter", "/.*/transport"]
```

Comment:
This change suggests that the `system_orchestrator` is managing the `transport` service for each transporter, is that the case? I can't find in the PR the place where this would happen

Reply:
Or is it instead that all the transporters run with the same configuration as the system orchestrator?